portal.go

```diff
@@ -921,6 +930,30 @@ func (portal *Portal) handleSignalDataMessage(source *User, sender *Puppet, msg
 	}
 }
 
+func (portal *Portal) catchUpHistory(source *User, fromRevision uint32, toRevision uint32, ts uint64) error {
+	log := portal.log.With().
+		Str("action", "catchUpHistory").
+		Stringer("source", source.MXID).
+		Uint32("from revision", fromRevision).
+		Uint32("to revision", toRevision).
```

Comment:
```suggestion
		Uint32("from_revision", fromRevision).
		Uint32("to_revision", toRevision).
```

log keys shouldn't have spaces